urbanaccess/gtfs/headways.py

```diff
@@ -129,10 +129,15 @@ def headways(gtfsfeeds_df,headway_timerange):
         route stop headways in units of minutes
         with relevant route and stop information
     """
+    
+    # TODO: Assertions in code during runtime should be handled in some other way.
```

Comment:
Agreed the assertions could be replaced with exceptions. We can address this later.